src/styles/foundation/_accessibility.scss

```diff
@@ -1,5 +1,7 @@
-@mixin high-contrast-outline {
+@mixin ms-high-contrast-outline($border-width: border-width()) {
```

Comment:
Love the addition of `$border-width` and the default value makes the change backwards compatible 🙆‍♂️ 

Curious about the name change though and wanted prompted the vendor prefix-esque addition? I'm not against it, but it does complicate the change a bit (you can see this in the need to edit files like `_buttons.scss`). 

It also means that this will need to be a major version bump since it will change the contract for any consumers that maybe using this mixin. On the other hand, it does make it clear that this is only adding applicable to `ms-` browsers. Which we may not want in the long run in case this mixin ever grows to include high contrast support for other browsers, but that thinking is a little premature here.

Reply:
This is a SUPER good point. There's a discussion happening in [slack](https://shopify.slack.com/archives/C4Y8N30KD/p1607455945447700), about this. 
@ben10code had a really important point that it is marked as non-standard according to[ MDN web docs.](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast). I'll wait to see what people say in the #accessibility channel from [this post](https://shopify.slack.com/archives/C03T74ZTD/p1607460676013100) from Benny.

Reply:
_**UPDATE_**: We will leave this mixin as is for this PR, and @ben10code will remove the media query and switch the color to transparent! 🥳

Reply:
@ben10code can you drop the `ms` prefix on the mixin in your PR to drop the media query? 

@alex-page just a heads up, if we cut a release of this as is it would likely need to be a major version bump since the renaming of the mixin would be a breaking change 🙈 but once @ben10code's changes are merge it will no longer be a breaking change 🤷‍♂️